string.c

```diff
@@ -11525,6 +11525,189 @@ rb_enc_interned_str_cstr(const char *ptr, rb_encoding *enc)
  *  without a ``!'' return a new String.  However, there are
  *  exceptions, such as String#[]=.
  *
+ *  == What's Here
+ *
+ *  === Methods for Creating a \String
+
+ *  - ::new:: Returns a new string.
+ *  - ::try_convert:: Returns a new string created from a given object.
+ *
+ *  === Methods for Setting \String State
+ *
+ *  - {#+string}[#method-i-2B-40]:: Returns a string that is not frozen:
+   *                                +self+, of not frozen; +self.dup+ otherwise.
+ *  - {#-string}[#method-i-2D-40]:: Returns a frozen copy of +self+.
```

Comment:
It returns self if already frozen.  Considering the documentation for `+@`, you should probably mention this.

Reply:
Fixed.